Rakefile

```diff
@@ -36,3 +36,64 @@ task :gen_nodeset do
   end
   puts nodeset
 end
+
+require 'rspec/core/rake_task'
+desc 'test tiering'
+RSpec::Core::RakeTask.new(:test_tier) do |t|
+
+  # Test mode must always be set
+  test_mode = ENV['BEAKER_TESTMODE'].to_sym
+  raise 'BEAKER_TESTMODE env variable must be either agent or apply.' unless %w(agent apply).include?(test_mode.to_s)
```

Comment:
Those were my thoughts as I was doing this, but I decided one thing at a time. Will get that done before the end of the sprint.